src/plugins/intel_cpu/tests/functional/single_layer_tests/classes/reduce.cpp

```diff
@@ -75,7 +84,18 @@ void ReduceCPULayerTest::SetUp() {
     std::vector<InputShape> inputShapes;
 
     std::tie(axes, opType, keepDims, reductionType, netPrecision, inPrc, outPrc, inputShapes) = basicParams;
-    inPrc = outPrc = netPrecision;
+    if (netPrecision == ElementType::boolean) {
```

Comment:
Why do we need to use precision hint here? Why net precision or model in/out precisions are not enough?

Reply:
Thanks Dmitry for the comment! Please correct me if I'm wrong. I think fp16 will be [converted](https://github.com/openvinotoolkit/openvino/blob/master/src/plugins/intel_cpu/src/transformations/transformation_pipeline.cpp#L220) to fp32 by the common transformation Pass ov::pass::ConvertPrecision. So precision hint is still needed to enforce fp16 on the side of CPU plugin. Further discussion is always welcomed. Thanks!

Reply:
Yeah, I missed that. Thanks for explanation. Lets use precision hint then.